packages/components/src/components/menu-flyout-list/menu-flyout-list.tsx

```diff
@@ -288,14 +290,22 @@ export class MenuFlyoutList {
 
   setPosition() {
     const { top, left } = this.triggerRect;
-    this.hostElement.style.top = `${top}px`;
-    this.hostElement.style.left = `${left}px`;
+    this.hostElement.style.left = `${left - 4}px`;
```

Comment:
is `${left - 4}px` also meant to affect user outside of the brand header?

<img width="536" alt="Captura de pantalla 2023-03-17 a les 12 36 43" src="https://user-images.githubusercontent.com/851105/225895376-fa7c10ae-b617-4911-8413-29909b9bb304.png">